bears/vcs/git/GitCommitBear.py

```diff
@@ -203,3 +239,85 @@ def check_body(self, body,
             yield Result(self, 'Body of HEAD commit contains too long lines. '
                                'Commit body lines should not exceed {} '
                                'characters.'.format(body_line_length))
+
+    def check_issue(self, body,
+                    body_regex: str=None,
```

Comment:
But this would affect the use of `body.splitlines()` :worried: Would cause me to join the lines again.